packages/celo-eslint/index.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    jest: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    'plugin:workspaces/recommended',
+  ],
+  plugins: ['import', 'unused-imports', 'jsx-a11y', 'prettier'],
+  rules: {
+    'prettier/prettier': 'warn',
+    'semi': 'off',
+    'import/no-anonymous-default-export': 'warn',
+    'unused-imports/no-unused-imports': 'warn',
+
+    // A11y
+    'jsx-a11y/alt-text': [
+      'warn',
+      {
+        elements: ['img'],
+        img: ['Image'],
+      },
+    ],
+    'jsx-a11y/aria-props': 'warn',
+    'jsx-a11y/aria-proptypes': 'warn',
+    'jsx-a11y/aria-unsupported-elements': 'warn',
+    'jsx-a11y/role-has-required-aria-props': 'warn',
+    'jsx-a11y/role-supports-aria-props': 'warn',
+
+    // Typescript
+    '@typescript-eslint/no-unused-vars': [
```

Comment:
I've had issues with `@typescript-eslint/no-unsafe-call` and`@typescript-eslint/no-unsafe-assignment`. Based solely on personal experience, I'd warn on them, but not sure if it's applicable to other people.

Reply:
Can you give an example of an issue?

Reply:
I went looking for it because I couldn't remember. It was when writing tests for example https://github.com/celo-org/packages-publisher/pull/10/files#file-src-cli-commands-publish-handler-test-ts-L221, the `expect.objectContaining` or other `expect.containing`.